core/src/raw/adapters/kv/backend.rs

```diff
@@ -87,6 +87,9 @@ impl<S: Adapter> Accessor for Backend<S> {
             cap.list_without_delimiter = true;
         }
 
+        cap.copy = true;
```

Comment:
`copy` should be enabled only when `read` & `write` has been set.

And `rename` needs `read`/`write`/`delete`.